general/documentation/installation.md

```diff
@@ -58,19 +58,23 @@ Gitpod is an alternative to local development and completely optional. We recomm
 
 ## Useful commands
 
-### Starting the development server
+### Starting the development server - the best way to edit docs locally
 
 ```console
 yarn start
 ```
 
+Once this completes, the docs will open in your browser. You can just edit the .md source files and reload to see the effect of your edits.
+
 ### Building the docs and serving them locally
 
 ```console
 yarn build
 yarn serve
 ```
 
+Not sure why you would want to do this if you are trying to write docs, but if you want to real a lot of docs locally this might be faster.
```

Comment:
Generally you don't. I'd say to remove this section entirely. I'm one of the few to use it regularly for debugging.